.github/workflows/ci.yaml

```diff
@@ -1,6 +1,7 @@
 ---
 env:
   NODEJS_VERSION: v18.18.2
+  RUN_TRIVY_SCAN: false 
```

Comment:
@zondervancalvez Does this mean that right now none of the scans are running by default? What I meant is that it should run for the production images (non-production images are the AIO images, anything with test in it's name, etc.)

Reply:
Addressed